addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -84,6 +99,12 @@ def create(self, values):
         return result
 
     def write(self, values):
+        # If it's a basic user then check if the timesheet is his own.
```

Comment:
Ideally, this should be separate PR?

Reply:
+ It would be great to make separate permissions to alter records of another user. Also, such implementation will limit extensibility for other modules?